packages/create-remix/templates/arc-stack/README.md

```diff
@@ -0,0 +1,72 @@
+# Remix Architect (AWS) Stack
+
+- [Remix Docs](https://remix.run/docs)
+
+## Architect Setup
+
+1. Globally install Architect and the AWS SDK
+
+```sh
+npm i -g @architect/architect aws-sdk
+```
+
+2. [Sign up][signup] and login to your AWS account
+   - To login with the CLI, you'll need to generate `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`, you can do so from your [security credentials][aws_access_key_id] and click on the "Access keys" tab, and then click "Create New Access Key", and then download and open the credentials file.
+   - Next, run `aws configure` and paste in your credentials.
+
+## Development
+
+```sh
+echo "NODE_ENV=development" >> .env
+echo SESSION_SECRET=$(openssl rand -hex 32) >> .env
+npm run dev
+```
+
+> If you don't have openssl installed, you can also use [1password][generate_password] to generate a random secret, just replace `$(openssl rand -hex 32)` with the generated secret.
+
+This starts your app in development mode, rebuilding assets on file changes.
+
+This is a pretty simple note-taking app, but it's a good example of how you can build a full stack app with Architect and Remix. The main functionality is creating users, logging in and out, and creating and deleting notes.
+
+### Relevant code:
+
+- creating users, and logging in and out [./app/models/user.server.ts](./app/models/user.server.ts)
+- user sessions, and verifying them [./app/session.server.ts](./app/session.server.ts)
+- creating, and deleting notes [./app/models/note.server.ts](./app/models/note.server.ts)
+
+The database that comes with `arc sandbox` is an in memory database, so if you restart the server, you'll lose your data. Production environments won't behave this way.
```

Comment:
This is a good caveat to mention about how sandbox emulates the db locally.

It might also be helpful to the user to mention that in staging and production on AWS, the database integration hooks up to DynamoDB so data will be persisted between Lambda executions and through new deployments.